specification/configuration.md

```diff
@@ -130,6 +136,27 @@ beyond the OpenTelemetry specification exist.
   - Distribution MUST default to `otlp`
   - Distribution MUST offer `jaeger-thrift-splunk` that defaults to `http://127.0.0.1:9080/v1/trace`
 
+### Real User Monitoring Libraries
+
+Real User Monitoring (RUM) instrumentation libraries cannot use environment variables for configuration.
+Instead, they MUST expose a `SplunkRum` object/class/namespace (depending on the language used) that allows
+setting the following properties (in a language-specific way: Java may use builders, Swift can use named
+parameters with default values, JavaScript can use objects, etc.):
+
+| Configuration property (default value) | Description                                                                                                                                                                                                 |
+| -------------------------------------- | -----------                                                                                                                                                                                                 |
+| `beaconUrl` ()                         | RUM beacon URL, ex. `https://rum-ingest.<realm>.signalfx.com/v1/rum`. [1]                                                                                                                                   |
```

Comment:
Implementations MUST enforce by default that beaconUrls are `https` only, and reject/fail to start otherwise.  Implementations MAY offer an `allowInsecureBeacon` option (default `false`) for customers wishing to use that.

Reply:
Does this need to be done by the library, though? Android does this by default, so I don't know that we should be doing it on top of the android way it works.

Reply:
The outcome should ideally be something the app developer can notice (an exception throw, error code returned, or error printout visible without additional config) the first time they integrate.  If Android is silently failing to send the wire traffic I'd bother to add the extra 3 lines of if/print/throw.

Reply:
No, it's not silent. Android will crash the app if you try it. :)

Reply:
The outcome is the same (app is failing), does it really matter if the check is built-in in the platform or if we do it ourselves?